### PR TITLE
YIP-6463 Added binaries and support for linux aarch64 platform

### PR DESCRIPTION
**Pull request by DiegoHenriquez-Youi**

### CODE REVIEW

**Review by bcturner — APPROVED** [review]
Looks good, but maybe wait for Casey or Patrick to approve

**Comment by patrickyoui** [comment]
Oh nice, we had added this to sprint too...

Did you need libshaderc.so? (I needed them for some reason, but maybe you found static'ly built ones?)

File: CMakeLists.txt [hunk 1]
```diff
@@ -15,8 +15,13 @@ if (CMAKE_CROSSCOMPILING)
         set(VULKAN_PREBUILT_BIN_GLSLC sdk/windows/Bin/glslc.exe)
         set(VULKAN_PREBUILT_BIN_SPIRV_CROSS sdk/windows/Bin/spirv-cross.exe)
     elseif (${CMAKE_HOST_SYSTEM_NAME} MATCHES "Linux")
-        set(VULKAN_PREBUILT_BIN_GLSLC sdk/linux/bin/glslc)
-        set(VULKAN_PREBUILT_BIN_SPIRV_CROSS sdk/linux/bin/spirv-cross)
+        if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
+            set(VULKAN_PREBUILT_BIN_GLSLC sdk/linux-aarch64/bin/glslc)
+            set(VULKAN_PREBUILT_BIN_SPIRV_CROSS sdk/linux/bin/spirv-cross)
```
Comment (patrickyoui) [inline CMakeLists.txt]: I think this path needs aarch64?

**Review by patrickyoui — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (DiegoHenriquez-Youi) [reply CMakeLists.txt]: you are correct. somehow missed that. thanks!

**Comment by DiegoHenriquez-Youi** [comment]
> Oh nice, we had added this to sprint too...
> 
> Did you need libshaderc.so? (I needed them for some reason, but maybe you found static'ly built ones?)

I didn't need it. And it built fine for me. I also checked if I had any libs installed for shaderc but I do not.

**Pull request merged**